From a0eb859d0ac42beb4120b306119d1f6955b528aa Mon Sep 17 00:00:00 2001
From: Oliver Neukum <oneukum@suse.com>
Date: Thu, 15 Mar 2018 20:49:17 +0100
Subject: [PATCH] Thunderbolt: kABI paddings added
Patch-mainline: Never (SUSE kABI padding)
References: bsn#129 bsc#1070681
Modified-by-SEL: Yes, modified due to different context

just in case

Signed-off-by: Oliver Neukum <oneukum@suse.com>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 include/linux/thunderbolt.h |    8 ++++++++
 1 file changed, 8 insertions(+)

--- a/include/linux/thunderbolt.h
+++ b/include/linux/thunderbolt.h
@@ -80,6 +80,9 @@ struct tb {
 	int index;
 	enum tb_security_level security_level;
 	size_t nboot_acl;
+
+	void *suse_kabi_padding;
+
 	unsigned long privdata[];
 };
 
@@ -104,6 +107,7 @@ static inline unsigned int tb_phy_port_f
 struct tb_property_dir {
 	const uuid_t *uuid;
 	struct list_head properties;
+	void *suse_kabi_padding;
 };
 
 enum tb_property_type {
@@ -137,6 +141,7 @@ struct tb_property {
 		char *text;
 		u32 immediate;
 	} value;
+	void *suse_kabi_padding;
 };
 
 struct tb_property_dir *tb_property_parse_dir(const u32 *block,
@@ -451,6 +456,8 @@ struct tb_nhi {
 	bool going_away;
 	struct work_struct interrupt_work;
 	u32 hop_count;
+
+	void *suse_kabi_padding;
 };
 
 /**
@@ -498,6 +505,7 @@ struct tb_ring {
 	u16 eof_mask;
 	void (*start_poll)(void *data);
 	void *poll_data;
+	void *suse_kabi_padding;
 };
 
 /* Leave ring interrupt enabled on suspend */
